Fastlane/Fastfile

```diff
@@ -190,3 +190,7 @@ lane :pr_changes_contains_path do |options|
 
   changes_contains_path
 end
+
+error do |lane, exception|
```

Comment:
Could you add the comment on top of the error handling I used to have in the project?
It might help others to figure out what this does ;)

`# This block will get executed when an error occurs, in any of the blocks (before_all, the lane itself or after_all).`

Reply:
```suggestion
# This block will get executed when an error occurs, in any of the blocks (before_all, the lane itself or after_all).
error do |lane, exception|
```